update_prevalence.py

```diff
@@ -1209,7 +1209,7 @@ def parse_jhu_vaccines_hourly_us(cache, data):
         try:
             state = data.get_state_or_raise(name=item.Province_State, country=item.Country_Region)
         except KeyError:
-            sentry_sdk.capture_message("Could not find state {item.Province_State}")
+            sentry_sdk.capture_message(f"Could not find state {item.Province_State}")
```

Comment:
This one was just a typo from my Sentry setup PR, so we weren't getting the names of the states not found. Once fixed I'll debug the actual issues